shared/login/register/code-page/index.render.native.js

```diff
@@ -29,6 +29,13 @@ class CodePageRender extends Component<void, Props, void> {
   }
 
   renderScanCode () {
+    if (this.props.qrCodeScanned) {
+      return (
+        <Box style={{...stylesQRScan, flex: 1, flexDirection: 'row', alignItems: 'center', justifyContent: 'center'}}>
+          <NativeActivityIndicator size='large' />
```

Comment:
we need something more descriptive than this. This makes it seem like its waiting for something like processing etc, its not clear you actually need to do something on a different computer